torch/csrc/Module.cpp

```diff
@@ -114,7 +113,6 @@ static PyObject * THPModule_initExtension(PyObject *_unused, PyObject *shm_manag
   torch::utils::initializeLayouts();
   torch::utils::initializeMemoryFormats();
   torch::utils::initializeQSchemes();
-  torch::utils::initializeDtypes();
```

Comment:
just to confirm- this function used to do the work of adding each dtype's name as an attribute on the module, and that's now done in `initDtypeBindings`. Technically that means that those attributes used to be set on the module as part of the `_initExtension()` method, vs. now they're set directly when the module is loaded?